pkg/crypto/helpers_test.go

```diff
@@ -0,0 +1,35 @@
+package crypto
+
+import (
+	"testing"
+)
+
+func TestTopicFromValidCert(t *testing.T) {
+	certificate, _ := ReadPEMCertificateFile("testdata/mock_push_cert.pem")
+	pushTopic, err := TopicFromCert(certificate)
+
+	if err != nil {
+		t.Errorf("fail %s", err)
+	}
+
+	if have, want := pushTopic, "com.apple.mgmt.External.18a16429-886b-41f1-9c30-2bd04ae4fc37"; have != want {
+		t.Errorf("have %d, want %d", have, want)
+	}
+}
+
+func TestTopicFromCertWithWrongUidPrefix(t *testing.T) {
+	expectPushCertError("testdata/mock_push_cert_wrong_uid_prefix.pem", t)
+}
+
+func TestTopicFromCertWithoutUid(t *testing.T) {
+	expectPushCertError("testdata/mock_push_cert_no_uid.pem", t)
+}
+
+func expectPushCertError(certPath string, t *testing.T) {
```

Comment:
helpers should have `t.Testing` as the first argument. let's reverse this order.

Reply:
Wilco